pkgs/development/python-modules/jax/default.nix

```diff
@@ -89,6 +89,8 @@ buildPythonPackage rec {
     "testKde3"
     "testKde5"
     "testKde6"
+    # Invokes python manually in a subprocess, which does not have the correct dependencies
```

Comment:
ooc what are the missing dependencies?

Reply:
It fails at importing `jaxlib.version` if I remember well.
Basically, I was able to succesfully execute the test artificially in `preCheck`. But the exact same snippet was failing in `pytest` when using `subprocess`.

Reply:
However, I checked (by `print`ing) that [this python](https://github.com/google/jax/blob/jaxlib-v0.4.24/tests/logging_test.py#L72) was exactly the same as the one I was using in the `preCheck`.

Reply:
must be something weird with pytest and subprocess then?

Reply:
nit: can we make this comment more specific? "does not have the correct dependencies" indicates to me that we should either identify the correct dependencies and use them, or specify what dependencies we are missing and why they can't be added

Reply:
Um sounds like upstream messing with PYTHONPATH and subprocesses